src/meta/api/src/schema_api_keys.rs

```diff
@@ -43,6 +44,7 @@ const PREFIX_TABLE_ID_LIST: &str = "__fd_table_id_list";
 const PREFIX_TABLE_COUNT: &str = "__fd_table_count";
 const PREFIX_DATABASE_ID_TO_NAME: &str = "__fd_database_id_to_name";
 const PREFIX_TABLE_ID_TO_NAME: &str = "__fd_table_id_to_name";
+const PREFIX_TABLE_STAGE_FILE: &str = "__fd_table_stage_file";
```

Comment:
This is not a table for `stage_file`. How about `copied_files`?

Reply:
ok